.github/workflows/unit-tests-check.yml

```diff
@@ -0,0 +1,39 @@
+name: Ensure Unit Tests Running
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  run_tests:
+    name: Test
+    runs-on: ubuntu-latest
```

Comment:
Let's make this explicitly 20.04 until we're ready to upgrade to something else.

Reply:
Done